rom/dev/tests/test_fmcalias_derivation.rs

```diff
@@ -53,3 +59,142 @@ fn test_firmware_gt_max_size() {
+
+    // Check PCR entry for AntiRollbackDisabled.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, AntiRollbackDisabled as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+    let anti_rollback_disable = hw.soc_ifc().fuse_anti_rollback_disable().read().dis();
+    assert_eq!(pcr_log_entry.pcr_data[0] != 0, anti_rollback_disable);
+
+    // Check PCR entry for VendorPubKeyHash.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, VendorPubKeyHash as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+    helpers::change_dword_endianess(vendor_pubkey_digest.as_bytes_mut());
+    assert_eq!(pcr_log_entry.pcr_data, vendor_pubkey_digest);
+
+    // Check PCR entry for OwnerPubKeyHash.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, OwnerPubKeyHash as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+    helpers::change_dword_endianess(owner_pubkey_digest.as_bytes_mut());
+    assert_eq!(pcr_log_entry.pcr_data, owner_pubkey_digest);
+
+    // Check PCR entry for VendorPubKeyIndex.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, VendorPubKeyIndex as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+    assert_eq!(pcr_log_entry.pcr_data[0], VENDOR_CONFIG_KEY_1.ecc_key_idx);
+
+    // Check PCR entry for FmcTci.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, FmcTci as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+
+    // Check PCR entry for FmcSvn.
+    pcr_log_entry_offset += core::mem::size_of::<PcrLogEntry>();
+    let pcr_log_entry =
+        PcrLogEntry::read_from_prefix(pcr_entry_arr[pcr_log_entry_offset..].as_bytes()).unwrap();
+    assert_eq!(pcr_log_entry.id, FmcSvn as u16);
+    assert_eq!(pcr_log_entry.metadata.as_bytes(), &[0u8; 4]);
+    assert_eq!(pcr_log_entry.pcr_id, 0);
+    assert_eq!(pcr_log_entry.pcr_data[0], FMC_SVN);
+}
```

Comment:
Would it be possible to have a test that hashes the data from the event log and ensures that it corresponds with the current value of the hardware PCRs?

Reply:
Yeah, should be doable.

Reply:
I'll add this as a separate change; Filed https://github.com/chipsalliance/caliptra-sw/issues/296 to track it.